examples/color_palette/src/main.rs

```diff
@@ -49,25 +49,25 @@ impl Sandbox for ColorPalette {
     fn update(&mut self, message: Message) {
         let srgb = match message {
             Message::RgbColorChanged(rgb) => palette::Srgb::from(rgb),
-            Message::HslColorChanged(hsl) => palette::Srgb::from(hsl),
-            Message::HsvColorChanged(hsv) => palette::Srgb::from(hsv),
-            Message::HwbColorChanged(hwb) => palette::Srgb::from(hwb),
-            Message::LabColorChanged(lab) => palette::Srgb::from(lab),
-            Message::LchColorChanged(lch) => palette::Srgb::from(lch),
+            Message::HslColorChanged(hsl) => palette::Srgb::from_color(hsl),
+            Message::HsvColorChanged(hsv) => palette::Srgb::from_color(hsv),
+            Message::HwbColorChanged(hwb) => palette::Srgb::from_color(hwb),
+            Message::LabColorChanged(lab) => palette::Srgb::from_color(lab),
+            Message::LchColorChanged(lch) => palette::Srgb::from_color(lch),
         };
 
-        self.theme = Theme::new(srgb.clamp());
+        self.theme = Theme::new(srgb);
```

Comment:
`.clamp` isn't needed anymore because `FromColor::from_color` clamps. Applicable doc comment:

```rs
/// Convert from T with values clamped to the color defined bounds.
///
/// ```
/// use palette::{Clamp, FromColor, Lch, Srgb};
///
/// let rgb = Srgb::from_color(Lch::new(50.0, 100.0, -175.0));
/// assert!(rgb.is_within_bounds());
/// ```
fn from_color(t: T) -> Self;
```